cmake/ArborXSettings.cmake.in

```diff
@@ -1,4 +1,6 @@
 set(ARBORX_ENABLE_ROCTHRUST @ARBORX_ENABLE_ROCTHRUST@)
 set(ARBORX_ENABLE_ONEDPL @ARBORX_ENABLE_ONEDPL@)
+set(ARBORX_CXX_COMPILER_BUNDLED_ONEDPL @ARBORX_CXX_COMPILER_BUNDLED_ONEDPL@)
```

Comment:
Why did you move it?

Reply:
Because we have all the other configuration variables here but I don't care much. Would rather have it moved back?

Reply:
The other are actual variables we export.  This one we define to use in the condition before unsetting it.
Please undo that change.

Reply:
Here you go.